src/alfrescoApi.ts

```diff
@@ -537,6 +537,10 @@ export class AlfrescoApi implements EventEmitter.Emitter {
         return this.config.authType === 'OAUTH';
     }
 
+    isPublicUrl(): boolean {
+         return this.oauth2Auth.isPublicUrl();
```

Comment:
Not sure why you need it can you do the same from outside?

Reply:
Good question. Regarding https://github.com/Alfresco/alfresco-ng2-components/pull/5363 I need to know for AuthGuardECM if that is a public url or not.